src/helpers/game_utils.ts

```diff
@@ -281,3 +283,73 @@ export async function getMatchingGroupNames(rawGroupNames: Array<string>, aliasA
 
     return result;
 }
+
+/**
+ * @param difficulty - The multiple choice difficulty
+ * @param guessMode - The guess mode
+ * @param gender - The correct answer's group's gender
+ * @param answer - The correct answer
+ * @param artistID - The correct answer's group's ID
+ * @returns three unshuffled incorrect choices based on difficulty
+ */
+export async function getThreeChoices(difficulty: GameType, guessMode: GuessModeType, gender: Gender, answer: string, artistID: number): Promise<[string, string, string]> {
+    let names: string[];
+    let sameArtistSong: string;
+    if (guessMode === GuessModeType.SONG_NAME || guessMode === GuessModeType.BOTH) {
+        switch (difficulty) {
+            case GameType.MC_EASY:
+                // Same gender as chosen artist
+                names = (await dbContext.kmq("available_songs").select("song_name")
+                    .where("members", gender)
+                    .andWhereNot("id_artist", artistID)).map((x) => x["song_name"]);
+                break;
+            case GameType.MC_MEDIUM:
+            {
+                // 2 same gender as chosen artist, 1 choice from chosen artist
+                const sameArtistSongList = await dbContext.kmq("available_songs").select("song_name")
+                    .where("id_artist", artistID)
+                    .andWhereNot("song_name", answer);
+
+                sameArtistSong = chooseRandom(sameArtistSongList)["song_name"];
```

Comment:
Can do a sort by rand() limit 1;

Reply:
I originally had that, I came across this article: http://jan.kneschke.de/projects/mysql/order-by-rand/